4funccalc.php

```diff
@@ -0,0 +1,50 @@
+<html>
+	<body>
+		<h1> Four Function Calculator </h1> <br> <br>
+			<form action=""
+				<p><input type="text" name="variable1"/></p>
+				<p>
+						<select name="math" id="math">
+							<option value='0'>--Select function--</option>
+							<option value='add'> + </option>
+							<option value='sub'> - </option>
```

Comment:
This is just a style preference, but I'd suggest using longer names instead of "sub" "mult" and "div". If someone were to read this code later on (and you'd be surprise how quickly you'll forget this yourself, so even if you read it a month from now this applies) they might not immediately realize the "div" means "divide"